registry/coder/modules/claude-code/main.tf

```diff
@@ -128,7 +128,7 @@ variable "claude_api_key" {
 
 variable "model" {
   type        = string
-  description = "Sets the model for the current session with an alias for the latest model (sonnet or opus) or a model’s full name."
+  description = "Sets the default model for Claude Code via ANTHROPIC_MODEL env var. If empty, Claude Code uses its default (Sonnet 4.5). Supports aliases (sonnet, opus) or full model names."
```

Comment:
I would suggest not using 4.5 in the description, as there may be a new 4.6 tomorrow.